tests/unittests/net/artifacts/photon_net_config/etc/systemd/network/10-cloud-init-vlan101.network

```diff
@@ -0,0 +1,17 @@
+[Address]
+Address=10.100.101.10/24
+
+[Link]
+MTUBytes=1400
+
+[Match]
+Name=vlan101
+
+[Network]
+DHCP=no
+DNS=1.1.1.1 9.9.9.9
+Domains=vlan101.local
+
+[Route]
+Destination=0.0.0.0/0
+Gateway=10.100.101.1
```

Comment:
There should also be a an additional section here that looks like:
```
[RoutingPolicyRule]
From=10.100.101.10
Table=101
```

Reply:
Sorry, my bad. Some mismatch in my yaml files. You are right. Thanks for the input. Will fix it.